src/briefcase/integrations/java.py

```diff
@@ -261,23 +257,18 @@ def install(self):
                 extract_dir=os.fsdecode(self.command.tools_path)
             )
         except (shutil.ReadError, EOFError):
-            raise BriefcaseCommandError(
-                """
+            raise BriefcaseCommandError(f"""
```

Comment:
I think some of the errors in this file may need the \ trick to avoid extra newlines (plus also auditing the newlines at the end of each message)